### DISCUSSION TITLE
/log-viewer not working

### DISCUSSION THREAD

**Discussion by kossa** [opcodesio/log-viewer#385] [General]
Hello,

I'm working with Laravel 11, and in my route I have this:
```
Route::get('{slug}', [FeedController::class, 'show'])->name('feeds.show')->where(['slug' => '.*']);
```

The issue is when I try: /log-viewer I got my url, how to prioritize the package route?

**Comment by arukompas** [comment]
@kossa I'm not sure that's a good practice to do a wildcard on the root level of the path. You'll run into similar issues when you want to include more features or packages into the system. For example, you might want to include Horizon or Nova at some point and you'll face the same issue.

I believe the best practice would be to put your feeds under a subpath, such as `/feed/{slug}`.

Alternatively, you can host Log Viewer behind a subdomain, e.g. `admin.example.com/log-viewer`.

Laravel does not provide a way to prioritise certain service providers over others. Unless maybe you can try manually adding `LogViewerServiceProvider` to your provider list **before** the app's `RouteServiceProvider`.

**Reply by kossa** [reply]
For `/feed/{slug}` I cannot bcz I'm showing directly some pages based on the slug(SEO)

In Laravel 11 there is no RouteServiceProvider https://github.com/laravel/laravel/tree/11.x/app/Providers
